lua/telescope/_extensions/file_browser/actions.lua

```diff
@@ -340,7 +344,7 @@ fb_actions.remove_file = function(prompt_bufnr)
         else
           fb_utils.delete_dir_buf(p:absolute())
         end
-        print(string.format("%s has been removed!", p:absolute()))
+        a.nvim_echo({ { string.format("\n%s has been removed!", p:absolute()) } }, false, {})
```

Comment:
I'm leaning towards switch printing / echoing to `vim.notify`. That way, users can have nicer notifications if they want to, and as I understand it (I'm not using it just yet), it's otherwise if not overridden by eg [nvim-notify](https://github.com/rcarriga/nvim-notify) practically equivalent to `nvim_echo`?

See also `:h nvim_notify`.

Thoughts?

Reply:
I'm ok with using `nvim_notify`, but I would suggest updating all instances of `nvim_echo` in the repo to use `nvim_notify` in a single commit, and then use `nvim_notify` from that point on. To keep it consistent instead of having echo in some places and notify in others.

That way we could also avoid nitpicking this API in future PRs. Any developers working on the project can see that we use nvim_notify and they would use it as well (hopefully, fingers crossed), I think that way we could be more efficient.

Reply:
Agreed.